client/my-sites/patterns/components/grid-gallery/style.scss

```
@@ -0,0 +1,51 @@
+@import "@automattic/components/src/styles/typography";
+@import "@wordpress/base-styles/breakpoints.scss";
+
+.patterns-grid-gallery {
+	display: flex;
+	flex-wrap: wrap;
```

Not a requirement, but wouldn't `display: grid` come in handy here?

---

Ofc both option work here, but specifically in this case - IMO grid is overkill. Flexbox is enough and it does exactly what it was created to do. So if you don't have objections - I would go with flexbox.

---

```suggestion
	display: grid;
	grid-template-columns: repeat(auto-fill, minmax(var(--item-min-width), 1fr));
```
I wouldn't say grid layout is "overkill"; I think we can achieve the same layout more succinctly. 

I couldn't help myself – I experimented and came up with this solution, which achieves the same layout we currently have!

With this solution, we can remove the `.patterns-grid-gallery__ghost-item` elements, which is nice.